rules_building_block/defense_evasion_suspicious_msiexec_execution.toml

```diff
@@ -18,7 +18,7 @@ references = [
 ]
 from = "now-119m"
 interval = "60m"
-index = ["logs-endpoint.events.*"]
+index = ["logs-endpoint.events.process-*"]
```

Comment:
```suggestion
index = ["logs-endpoint.events.process-*", "logs-system.security*", "winlogbeat-*", "logs-windows.*", "endgame-*"]
```

Reply:
Edited this one, as `process.args_count` is available on Wineventlogs